pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -199,6 +204,31 @@ func RewriteNodeIPs(nodeList []*corev1.Node) *metadataOptions {
 	}
 }
 
+func RewriteBootstrapIPs(bootstrapIP string) *metadataOptions {
+	return &metadataOptions{
+		rewriteSecretFn: func(secret *corev1.Secret) {
+			if secret.Namespace != "openshift-etcd" {
```

Comment:
unnecessary restriction.

Reply:
Other components may use `auth.openshift.io/certificate-hostnames` annotation differently, perhaps we should rename `RewriteBootstrapIPsInEtcdSecrets`?